RecoParticleFlow/PFClusterProducer/python/particleFlowZeroSuppressionECAL_cff.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+
+pfZeroSuppressionThresholds_EB = [0.250]*170
+pfZeroSuppressionThresholds_EEminus = [1.25023,   1.25033,   1.25047,   1.25068,   1.25097,   1.25139,   1.25199,   1.25286,   1.2541,   1.25587,  # rings 170-179 (EE-) / 209-218 (EE+) 
+                                       1.25842,   1.26207,   1.26729,   1.27479,   1.28553,   1.30092,   1.32299,   1.35462,   1.39995,  1.46493,  # rings 180-189 (EE-) / 219-228 (EE+)
+                                       1.55807,   1.69156,   1.88291,   2.15716,   2.55027,   3.11371,   3.92131,   5.07887,   6.73803,  9.11615, #
+                                       10,   10,   10,   10,   10,   10,   10,   10,   10 ]
```

Comment:
thanks @slava77 . Done.
I tested it with 2016 era and 2016 data and takes the right thresholds. I tried with 2017, but I cannot do on 2016 data (takes the new geometry and complains), but should be OK.